kotlin/services/lambda/src/main/kotlin/com/kotlin/lambda/LambdaScenario.kt

```diff
@@ -0,0 +1,218 @@
+// snippet-sourcedescription:[LambdaScenario.kt demonstrates how to perform various operations by using the LambdaClient object.]
+// snippet-keyword:[AWS SDK for Kotlin]
+// snippet-keyword:[AWS Lambda]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[04/14/2022]
+// snippet-sourceauthor:[AWS-scmacdon]
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.kotlin.lambda
+
+// snippet-start:[lambda.kotlin.scenario.import]
+import aws.sdk.kotlin.services.lambda.LambdaClient
+import aws.sdk.kotlin.services.lambda.model.CreateFunctionRequest
+import aws.sdk.kotlin.services.lambda.model.DeleteFunctionRequest
+import aws.sdk.kotlin.services.lambda.model.FunctionCode
+import aws.sdk.kotlin.services.lambda.model.GetFunctionRequest
+import aws.sdk.kotlin.services.lambda.model.InvokeRequest
+import aws.sdk.kotlin.services.lambda.model.ListFunctionsRequest
+import aws.sdk.kotlin.services.lambda.model.Runtime
+import aws.sdk.kotlin.services.lambda.model.UpdateFunctionCodeRequest
+import aws.sdk.kotlin.services.lambda.model.UpdateFunctionConfigurationRequest
+import kotlinx.coroutines.delay
+import kotlin.system.exitProcess
+// snippet-end:[lambda.kotlin.scenario.import]
+
+/**
+ Before running this code example, set up your development environment, including your credentials.
+
+ For more information, see this documentation topic:
+
+ https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/setup.html
+
+ This example performs these tasks.

```

Comment:
This example performs the following tasks:

Reply:
fixed